pkgs/by-name/js/jspecify/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchMavenArtifact,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jspecify";
+  version = "1.0.0";
+
+  src = fetchMavenArtifact {
+    groupId = "org.jspecify";
+    artifactId = "jspecify";
+    inherit version;
+    hash = "sha256-H61ua+dVd4Hk0zcp1Jrhzcj92m/kd7sMxozjUer9+6s=";
+  };
+
+  installPhase = ''
+    install -m444 -D ${src}/share/java/*${pname}-${version}.jar "$out/share/java/${pname}-${version}.jar"
```

Comment:
We have a shortcut:
```diff
diff --git a/pkgs/by-name/js/jspecify/package.nix b/pkgs/by-name/js/jspecify/package.nix
index 4c240ac660da..dfcad28109b9 100644
--- a/pkgs/by-name/js/jspecify/package.nix
+++ b/pkgs/by-name/js/jspecify/package.nix
@@ -16,7 +16,9 @@ stdenv.mkDerivation rec {
   };
 
   installPhase = ''
-    install -m444 -D ${src}/share/java/*${pname}-${version}.jar "$out/share/java/${pname}-${version}.jar"
+    runHook preInstall
+    install -Dm444 "${src.jar}" "$out/share/java/${pname}-${version}.jar"
+    runHook postInstall
   '';
 
   meta = {
```

Reply:
Fixed, amended, and force-pushed.

Reply:
Missing the middle part?

Reply:
> Missing the middle part?

Missing the whole change -- I pushed the update to the wrong remote repo. Sorry for the confusion.

Reply:
OK, this time I pushed it to the right place (and verified in the GitHub UI.)